cypress/e2e/loans/renewal-success.cy.js

```diff
@@ -159,12 +159,10 @@ describe('Renewal', () => {
     CheckinActions.checkinItemViaApi({
```

Comment:
please fix issue 'moment' is defined but never used. Allowed unused vars must match /React/u